data/ka'het/ka'het missions.txt

```diff
@@ -41,6 +41,7 @@ mission "First Contact: Ka'het: Remnant 1A"
 		government "Remnant"
 	to offer
 		has "Graveyard Label: offered"
+		has "First Contact: Remnant: declined"
```

Comment:
What you should actually do is actually have the correct condition for the map label. :)
It should be `has "event: ember waste label"`. Using `has "remnant blood test pure"` would work as well and be more "correct" in terms of the intention of that condition.

This change also needs made for the other Remnant mission here.

Reply:
..but then, how do you know that the player visited the graveyard?

Reply:
Done